packages/one-app-runner/__tests__/bin/__snapshots__/one-app-runner.spec.js.snap

```diff
@@ -338,36 +275,20 @@ exports[`command errors out if an unknown option is given 1`] = `
 Array [
   Array [
     "Options:
-  --version                 Show version number                        [boolean]
-  --module-map-url          module map for One App to use and fetch modules from
-                                                                        [string]
-  --root-module-name        name of the module to serve as an entry point to
-                            your application                 [string] [required]
-  --docker-image            docker image to use for One App  [string] [required]
-  --parrot-middleware       path to parrot dev middleware file for One App to
-                            use for Parrot mocking                      [string]
-  --modules                 path to local module to serve to One App     [array]
-  --dev-endpoints           path to dev endpoints file for One App to use for
-                            its One App Dev Proxy set up                [string]
-  --envVars                 Environment variables to be applied to the One App
-                            instance
-  --output-file             File to redirect all stdout and stderr from One App
-                            Container to                                [string]
-  --create-docker-network   Creates a new docker network
-                                                      [boolean] [default: false]
-  --docker-network-to-join  Connect One App container to a docker network
-                                                                        [string]
-  --use-host                Use req.headers.host instead of localhost for
-                            one-app-dev-cdn           [boolean] [default: false]
-  --offline                 skip docker pull when the docker registry is not
-                            available / offline       [boolean] [default: false]
-  ----container-name        Assign a container name with the --name option
+  --help               Show help                                       [boolean]
+  --version            Show version number                             [boolean]
+  --module-map-url     module map for One App to use and fetch modules from
                                                                         [string]
-  --help                    Show help                                  [boolean]",
+  --root-module-name   name of the module to serve as an entry point to your
+                       application                           [string] [required]
+  --docker-image       docker image to use for One App       [string] [required]
+  --parrot-middleware  path to parrot dev middleware file for One App to use for
+                       Parrot mocking                        [string] [required]",
   ],
   Array [],
   Array [
-    "Unknown arguments: not-a-valid-option, notAValidOption",
```

Comment:
This snapshot has changed with a different value, I assume it is because the new rule fails fast for `--parrot-middleware` before yargs sees the `invalid arguments` 🤔

Reply:
might become a DX issue if not all arguments are printed in the "help" when an invalid argument is added (typo for example)

Reply:
@infoxicator yeah was trying to figure out a way to get the tests to pass but maintain the same functionality.